queries/scala/highlights.scm

```diff
@@ -0,0 +1,52 @@
+;; keywords
+
+[
+  "abstract"
+  "case"
```

Comment:
Case might be `conditional`

Reply:
what about `case object Blah extends Meh`. `case` is not conditional in that situation.

Reply:
I put a special situation for `case` as a conditional at the bottom so it overrides the default situation `case class`

Reply:
You don't have to add it if it doesn't fit. I don't know Scala.